docker-compose.yml

```diff
@@ -4,6 +4,13 @@
 
 version: '2'
 services:
+  build-requirements:
```

Comment:
I see this is very similar to the one I added to Lando API's docker-compose, but it does not use a build target (which makes this go much faster since it only needs to use the Python base image in that case).

Also just thinking now that for both of them, it might be better to use `Dockerfile` (with a build target) and not `Dockerfile-dev` since the requirements always need to match production (would be nice if we could have a single source for the python version though).

For the purposes of this PR though I think specifying a target is sufficient.

Reply:
Good call, switched to using the prod Dockerfile and used the pre-existing `assets` target.

Having both Dockerfiles pull from a base image with the Python version defined would be nice to have, I agree. After this switch the requirements building is much faster (I have another patch coming after this one to update a package).

Sorry in advance but I amended and force pushed instead of using a fixup commit. Hopefully this change is small enough that it's fine. :)